src/content/development/contribution-guide/_index.en.md

```diff
@@ -106,7 +106,21 @@ To be expanded
 
 #### General Availability
 
-To be expanded
+Once a release candidate is deemed stable and has no blocker bugs it will be released for general availability.
+Prior to releasing, the release manager will verify that there have been no changes on the stable branches since the last release
```

Comment:
This release, and I think most/all releases so far, we've blocked changes that touched code or dependencies of code, but not CI (and I guess docs would be the same if we edited branched-docs). As is this is worded, it would change that practice. Is that what we want?

Reply:
We're talking about the stable brances, yes?
We had the same policy before more or less, it's now formal.

The website doesn't use branches so it's not affected, and CI changes backports to stable branches shouldn't be exempt automatically.

Reply:
Okay that makes sense, thanks. I was thinking of the time we're basically "code frozen" right before cutting stable branches.